lib/server-render.js

```diff
@@ -66,7 +67,7 @@ function serverRender (entry, cb) {
     }
   }
 
-  cb(null, render)
+  cb(null, render, ssrError)
```

Comment:
is there a reason the error can't be passed in the first parameter?

Reply:
Agree — think it'd be nice if `error` had an extra property on it so we can figure out what type of error it is (pretty multiline one, regular unknown error, SSR error) and we proceed from there. Prevents from adding more parameters in the future too, as we expand our error handling capabilities (:

Reply:
makes total sense! I was overthinking this :)